pkg/collector/upgrade/v0_24_0_test.go

```diff
@@ -57,7 +57,7 @@ func TestHealthCheckEndpointMigration(t *testing.T) {
 
 	// verify
 	assert.Equal(t, `extensions:
-  health_check: null
```

Comment:
@jpkrohling the issue of `null` existed in the past too. The other contributor ignored it by adding `null` during the validation as the parser injects null to empty value.

Reply:
Can you please create an issue to track this?